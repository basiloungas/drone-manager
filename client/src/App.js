import React from 'react';
import { Layout } from 'antd';

import DroneList from './pages/dronesList';

const { Header, Sider, Content } = Layout;

const App = () => {
  return (
    <Layout>
      <Header>
        <h1> Welcome to the Drone Manager</h1>
      </Header>
      <Layout>
        <Sider />
        <Content>
          <DroneList />
        </Content>
        <Sider />
      </Layout>
    </Layout>
  );
};

export default App;
