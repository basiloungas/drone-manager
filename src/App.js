import React from 'react';

import Layout from './components/layout';
import DroneList from './pages/dronesList';

const App = () => {
  return (
    <Layout title="Welcome to the Drone Manager">
      <DroneList />
    </Layout>
  );
};

export default App;