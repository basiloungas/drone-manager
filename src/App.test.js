import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

import Layout from './components/layout';
import DroneList from './pages/dronesList';

describe('App', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<App />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });

  test('renders layout with "title"', () => {
    const wrapper = shallow(<App />);

    expect(wrapper.find(Layout).prop('title')).toEqual('Welcome to the Drone Manager');
  });

  test('renders "DroneList" component', () => {
    const wrapper = shallow(<App />);

    expect(wrapper.find(DroneList).exists()).toBe(true);
  });
});
