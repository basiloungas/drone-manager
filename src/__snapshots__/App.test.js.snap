// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<App
  title="Welcome to the Drone Manager"
>
  <Connect(Homepage) />
</App>
`;
